docs/cody/clients/feature-reference.mdx

```diff
@@ -4,28 +4,27 @@
-| Edit code                                | ✅           | ✅             | ❌                    | ❌      |
-| Generate unit test                       | ✅           | ✅             | ❌                    | ❌      |
-| Ask Cody to Fix                          | ✅           | ✅             | ❌                    | ❌      |
+|               **Feature**                | **VS Code** | **JetBrains** |       **Web**        | **Visual Studio** | **CLI** |
+| ---------------------------------------- | ----------- | ------------- | -------------------- | ----------------- | ------- |
+| Chat with Cody                           | ✅           | ✅             | ✅                    | ✅                 | ✅       |
+| Chat history                             | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| Clear chat history                       | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| Edit sent messages                       | ✅           | ❌             | ✅                    | ✅                 | ❌       |
+| Show context files                       | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| @-file                                   | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| @-symbol                                 | ✅           | ❌             | ✅                    | ✅                 | ❌       |
+| Ollama support (experimental)            | ✅           | ✅             | ❌                    | ✅                 | ❌       |
+| LLM Selection                            | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| **Context Selection**                    |             |               |                      |                   |         |
+| Single-repo context                      | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| Multi-repo context                       | ❌           | ❌             | ✅ (public code only) | ❌                 | ❌       |
+| Local context                            | ✅           | ✅             | ❌                    | ✅                 | ✅       |
+| OpenCtx context providers (experimental) | ✅           | ❌             | ❌                    | ❌                 | ❌       |
+| **Prompts and Commands**                 |             |               |                      |                   |         |
+| Access to prompts and Prompt library     | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| Edit code                                | ✅           | ✅             | ❌                    | ✅                 | ❌       |
+| Generate unit test                       | ✅           | ✅             | ❌                    | ✅                 | ❌       |
```

Comment:
```suggestion
| Generate unit test                       | ✅           | ✅             | ❌                    | ❌                 | ❌       |
```

Reply:
Yeah this will go away with Nov 25th release.